core/server/data/migrations/versions/4.0/14-remove-orphaned-stripe-records.js

```diff
@@ -0,0 +1,35 @@
+const {createIrreversibleMigration} = require('../../utils');
+const logging = require('../../../../../shared/logging');
+
+module.exports = createIrreversibleMigration(async function up(connection) {
+    if (connection.client.config.client === 'mysql') {
+        logging.info('Skipping removal of orphaned stripe records for MySQL');
+        return;
+    }
+
+    logging.info('Removed orphaned rows from members_stripe_customers');
```

Comment:
nitpick: you might want to say `removing` instead of `removed`, so that if an errrr happens after the log, users understand the action hasn't finished successfully yet.

Reply:
Good call! :relaxed: